source/GameInterface/DynamicSync/DynamicHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Common.Messaging;
+using Common.Network;
+using GameInterface.Services.ObjectManager;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GameInterface.DynamicSync
+{
+    public class DynamicHandler : IHandler
```

Comment:
Have this be it's own type, this is technically not a handler

```suggestion
    public class DynamicHandlerCollection : IDynamicHandlerCollection
```